client/go/src/github.com/vespa-engine/vespa/utils/print.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// Print functions for color-coded text.
+// Author: bratseth
+
+package utils
+
+import (
+    "fmt"
+    "io"
+    "os"
+)
+
+// Set this to have output written somewhere else than os.Stdout
+var Out io.Writer
+
+func init() {
+    Out = os.Stdout
+}
+
+// Prints in default color
+func Print(messages ...string) {
+    print("", messages)
+}
+
+// Prints in a color appropriate for errors
+func Error(messages ...string) {
+    print("\033[31m", messages)
+}
+
+// Prints in a color appropriate for success messages
+func Success(messages ...string) {
+    print("\033[32m", messages)
```

Comment:
Yes, we need a script mode. I think the default should focus on usability though, since that's the main use case and I don't really think people should use this for scripting (not that that will stop them).